test/unit/chatCommands.test.ts

```diff
@@ -238,6 +260,46 @@ suite("Chat Commands Tests", () => {
             expect(mockMainController.onChooseDatabase).to.have.been.calledOnce;
         });
 
+        test("help command shows capabilities and button when disconnected", async () => {
+            const chatRequest = createChatRequest("help");
+            mockConnectionManager.getConnectionInfo.returns(undefined as unknown as ConnectionInfo);
+
+            const result = await handleChatCommand(
+                chatRequest,
+                chatStream,
+                mockMainController as unknown as MainController,
+                undefined, // No connection URI
+            );
+
+            expect(result.handled).to.be.true;
+            expect(result.errorMessage).to.be.undefined;
+            // Should show welcome message and all capabilities
+            expect(chatStreamMarkdownStub).to.have.been.called;
+            // Should show connection button since not connected
+            expect(chatStreamButtonStub).to.have.been.calledOnce;
+            const buttonCall = chatStreamButtonStub.getCall(0);
+            expect(buttonCall.args[0].command).to.equal("mssql.copilot.newQueryWithConnection");
```

Comment:
I think there's a `to.have.been.calledWith()` helper that makes this more fluent and improves the failure message default.

Reply:
updated to use `to.have.been.calledOnceWith`